package.json

```diff
@@ -233,6 +238,12 @@
           "group": "2_powershell"
         }
       ],
+      "editor/title/context": [
+        {
+          "when": "resourceExtname  == .ps1",
```

Comment:
Does this work:
```
"when":  "resourceFilename =~ /\.[Tt]ests\.ps1$/"
```

Reply:
Thanks for the tip with the regex, I did not know this was supported until now, I tried using `/\\.[Tt]ests\\.ps1$/` to escape the backslash in JSON but it did not work. Neither did `=~ /.*tests\\.ps1$/` work as proposed [here](https://github.com/Microsoft/vscode/issues/66461#issuecomment-453879835) by the vscode team. I suppose I have to play with this a bit more but thanks so far.

Reply:
Thanks, I had to escape the backslash in JSON and then your suggestion worked: `"resourceFilename =~ /\\.[Tt]ests\\.ps1$/"`